src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -19,7 +26,14 @@
      *                 В любом другом случае бросить DateTimeException
      **/
     public String todayDate(DateTimePart datePart) {
-        return "Today";
+        LocalDate localDate = LocalDate.now();
+        switch (datePart) {
+            case DAY: return String.valueOf(localDate.getDayOfMonth());
```

Comment:
why do we use in different cases method `String.valueOf` and `toString()`? Let's use one of them

Reply:
different cases return different type of data)